packages/deployment/lib/configValidators.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const get = require('lodash.get');
+/**
```

Comment:
Does ESlint not complain about extra lines between the imports and the docstring?